src/i18n/en-events.ts

```diff
@@ -8,6 +8,7 @@ const eventsMessageDescriptors = {
       ${ColonyAndExtensionsEvents.ColonyFundsMovedBetweenFundingPots} {{initiator} transferred {amount} {tokenSymbol} from the {fromDomain} to the {toDomain}}
       ${ColonyAndExtensionsEvents.TokensMinted} {{initiator} minted {amount} {tokenSymbol} to {recipient}}
       ${ColonyAndExtensionsEvents.DomainAdded} {{initiator} added Team: {fromDomain}}
+      ${ColonyAndExtensionsEvents.DomainMetadata} {{initiator} changed {fromDomain} teams's from}
```

Comment:
Is it used anywhere? It doesn't seem to be complete but I couldn't find it in this form anywhere in the app.

Reply:
nee, need to remove it, old forgot thing 😆  good catch